src/components/Agencies/Agencies.tsx

```diff
@@ -1,3 +1,13 @@
+import { Container, Grid, Paper } from '@material-ui/core';
 import React from 'react';
 
-export const Agencies = () => <div>Agencies</div>;
+export const Agencies = () => (
+  <>
+    <h1>Agencies</h1>
```

Comment:
title should be a separate component in /shared/components and can be used in other components